src/test/run-make/issue-40535/Makefile

```diff
@@ -3,8 +3,8 @@
 # * `bar` declares and depends on `extern crate baz`
 # * All crates built in metadata-only mode (`cargo check`)
 all:
-	$(RUSTC) baz.rs --emit=metadata
-	$(RUSTC) bar.rs --emit=metadata --extern baz=libbaz.rmeta
-	$(RUSTC) foo.rs --emit=metadata -L dependency=. --extern bar=libbar.rmeta 2>&1 | \
+	$(RUSTC) baz.rs --emit=metadata --out-dir=$(TMPDIR)
```

Comment:
`--out-dir=$(TMPDIR)` is already included in [`$(RUSTC)`](https://github.com/abonander/rust/blob/3cbdb8f8d95e561ddbebba5940be7f838290af47/src/test/run-make/tools.mk#L9), would be very strange if adding it again here made a difference...